front/pages/api/auth/[auth0].ts

```diff
@@ -28,10 +31,18 @@ export default handleAuth({
     };
 
     // Set the Auth0 connection based on the provided connection param, redirecting the user to the correct screen.
-    if (connection) {
+    if (isString(connection)) {
       defaultAuthorizationParams.connection = connection;
     }
 
+    if (isString(screen_hint) && screen_hint === "signup") {
+      defaultAuthorizationParams.screen_hint = screen_hint;
+    }
+
+    if (isString(login_hint)) {
+      defaultAuthorizationParams.login_hint = login_hint;
```

Comment:
Shouldn't this be validated before bring passed to auth0 ?

Reply:
Yep was still WIP. FWIW Auth0 is pretty permissive here. Added a check on valid email address.